src/category_theory/monoidal/category.lean

```diff
@@ -284,24 +284,32 @@ begin
 end
 
 -- See Proposition 2.2.4 of <http://www-math.mit.edu/~etingof/egnobookfinal.pdf>
-@[simp] lemma left_unitor_tensor (X Y : C) :
-  ((α_ (𝟙_ C) X Y).hom) ≫ ((λ_ (X ⊗ Y)).hom) =
-    ((λ_ X).hom ⊗ (𝟙 Y)) :=
+lemma left_unitor_tensor' (X Y : C) :
+  ((α_ (𝟙_ C) X Y).hom) ≫ ((λ_ (X ⊗ Y)).hom) = ((λ_ X).hom ⊗ (𝟙 Y)) :=
 by rw [←tensor_left_iff, id_tensor_comp, left_unitor_product_aux]
 
-@[simp] lemma left_unitor_tensor_inv (X Y : C) :
-  ((λ_ (X ⊗ Y)).inv) ≫ ((α_ (𝟙_ C) X Y).inv) =
-    ((λ_ X).inv ⊗ (𝟙 Y)) :=
+@[simp]
+lemma left_unitor_tensor (X Y : C) :
+  ((λ_ (X ⊗ Y)).hom) = ((α_ (𝟙_ C) X Y).inv) ≫ ((λ_ X).hom ⊗ (𝟙 Y)) :=
```

Comment:
Here I've reconfigured the `@[simp]` lemmas so they can unpack a unitor for a tensor product, in terms of other morphisms.

Reply:
Could you add this comment to the first post in the PR? Worth preserving in the commit history.

Reply:
Done.